.pylintrc

```diff
@@ -75,3 +75,9 @@ disable=
 	# NOTE(nathaniel): I have disputed the premise of this inspection from
 	# the beginning and will continue to do so until it goes away for good.
 	useless-else-on-loop,
+	# NOTE(mmx): Our existing code is written to favor explicit else over
```

Comment:
This sounds rather arbitrary; let's put it on a more solid footing:
```
NOTE(nathaniel): A single statement that always returns program
control is better than two statements the first of which sometimes
returns program control and the second of which always returns
program control. Probably generally, but definitely in the cases of
if:/else: and for:/else:.
```
. Note that this statement also covers the immediately-above suppression of `useless-else-on-loop`; the previous `NOTE` on `useless-else-on-loop` could go away were this `NOTE` attached to both suppressions.